ui/web-v2/apps/admin/src/pages/feature/targeting.tsx

```diff
@@ -227,13 +253,14 @@ export const FeatureTargetingPage: FC<FeatureTargetingPageProps> = memo(
       );
     }, [dispatch, currentEnvironment]);
 
-    if (isLoading) {
-      return (
-        <div className="p-9 bg-gray-100">
-          <DetailSkeleton />
-        </div>
-      );
-    }
+    // if (isLoading) {
```

Comment:
Is there a reason to comment this code?

Reply:
I will remove this. We don't need this anymore.